_includes/contributor-list2.html

```diff
@@ -24,6 +24,14 @@
 	</tr>
 	{% endif %}
 
+	{% if include.contributors.data %}
```

Comment:
it's a nice idea, we'll have to make some changes in other places to support it though

Reply:
https://github.com/galaxyproject/training-material/commit/417be6409174fc5510d942759e2f63c0a425d7a4 and https://github.com/galaxyproject/training-material/commit/c61df5bb6c4022b1e0ee7c29786ca9946cf361e4: Thank you, looks good to me, although I'm experienced with these templates.